sdcm/tester.py

```diff
@@ -782,19 +783,39 @@ def init_resources(self, loader_info=None, db_info=None,
         elif cluster_backend == 'baremetal':
             self.get_cluster_baremetal()
 
+    def set_seeds(self):
+        seeds_selector = self.params.get('seeds_selector', default='first')
```

Comment:
avoid defaults in the code, they should be in YAML or SCT config

Reply:
So we need to change it for all parameters

Reply:
done